fixtures/fixtures.js

```diff
@@ -55,12 +55,15 @@ var Fixtures = {
         ]
     },
     voucherStatus: {
-        "success":
-            {"success":["RVNT12345678"], "fail":[], "invalid":[]},
-        "fail":
-            {"success":[], "fail":["FAL11111111"], "invalid":[]},
-        "invalid":
-            {"success":[], "fail":[], "invalid":["INV1"]},
+        "success": [{"message": 'Voucher is valid'}],
+        "warning": [
```

Comment:
Where did the text for these come from? It does not match the ones in: https://github.com/neontribe/ARCVService/pull/102

I am worried that copy will slip through the cracks.